packages/main/src/ToolbarItem.ts

```diff
@@ -0,0 +1,79 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { TemplateFunction } from "@ui5/webcomponents-base/dist/renderer/executeTemplate.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+
+import ToolbarItemOverflowBehavior from "./types/ToolbarItemOverflowBehavior.js";
+
+@customElement({
+	tag: "ui5-tb-item",
```

Comment:
It's strange for an abstract class to have a tag. This component can never be instantiated. Can the tag be skipped?